goldstone/core/tasks.py

```diff
@@ -150,3 +151,55 @@ def log_event_search():
 
     cadf = CADFEventDocType(event=e)
     return cadf.save()
+
+
+@celery_app.task()
+def check_for_pending_alerts():
+    """
+     Run an AlertSearch query to check for any pending alerts to be fired.
+    """
+
+    saved_alerts = AlertSearch.objects.all()
+
+    for obj in saved_alerts:
+        # execute the search, and assuming no error, update the last_ times
+        s, start, end = obj.search_recent()
+        obj.last_start = start
```

Comment:
probably want to defer saving until after you get past the search execution.  if execute() throws an exception, we probably want to reprocess that time span to ensure that we don't miss an alert generation.